fix: Unresolved dependencies

Tommylans opened the pull request:
The current imports failed to resolve to a Golang package.

I updated the imports to the new places like they described on the update page https://periph.io/news/2020/a_new_start/ .

MichaelS11 commented:
Besides the spelling error, not sure what these changes are fixing. Are go.mod and go.sum required for go-dht to work?

Tommylans commented:
Oh sorry I forgot to add the description. The go.sum and go.mod weren't required. I also changed the package imports, the the original ones were deprecated and not working anymore. So the command `go get github.com/MichaelS11/go-dht` fails. 

I updated the imports to the new places like they described on the update page https://periph.io/news/2020/a_new_start/ .

And it should be working again after this edit.

MichaelS11 commented on dht.go:
```diff
@@ -2,9 +2,8 @@ package dht
 
 import (
 	"fmt"
+	"periph.io/x/host/v3"
```
Please keep external imports at the end.

Tommylans replied:
This does not comply with the Golang coding standard formatting.

MichaelS11 replied:
I believe if you leave the new line between, the Go formatting will leave the newline there.

Tommylans replied:
I didn't know that existed in Go. I applied your suggested changes.

MichaelS11 commented on dhtNotWindows.go:
```diff
@@ -1,15 +1,15 @@
+//go:build !windows
```
I do not think this line is needed, but please educate me if it is.

Tommylans replied:
This is needed because it will say to the compiler to not compile this on Windows. Which also is noted in the filename.

MichaelS11 replied:
Doesn't the line below do the same thing?
// +build !windows

Tommylans replied:
It wasn't necessary to have both so I removed the +build windows

MichaelS11 replied:
Looks like the syntax has changed. Please leave the old one and add the new one.
Also please add the new format for dhtWindows.go as well.

"Go versions 1.16 and earlier used a different syntax for build constraints, with a "// +build" prefix. The gofmt command will add an equivalent //go:build constraint when encountering the older syntax."
https://pkg.go.dev/cmd/go#hdr-Build_constraints

Tommylans replied:
Added both syntaxes 👍

Tommylans commented:
Oh btw just know that the package isn't working for the people that try to install it.

MichaelS11 commented:
Looks good, thank you very much.
